### DISCUSSION TITLE
Differences between curl and polar wake models in FAST Farm for a fixed bottom farm

### DISCUSSION THREAD

**Discussion by MYMahfouz** [OpenFAST/openfast#1756] [Q&A] [answered]
Dear all,

I am seeing differences between the curl model and the polar model results for FAST.Farm for a turbine in the wake. The difference is around `1m/s`. 

Is this expected? What is the reason behind this difference.

Below if a figure of the rotor average wind speed for the polar model in orange and the curl model in blue.

Thanks in advance.

Best regards,
Youssef

![image](https://github.com/OpenFAST/openfast/assets/33868271/66850f52-e95f-4b2e-ab6d-f3adbadde864)

**Comment by jjonkman** [comment]
Dear @MYMahfouz,

The formulation of the curled wake model is different than for the polar model, so, I wouldn't expect identical results even for a rotor without skewed inflow.  In our curled wake publication (https://onlinelibrary.wiley.com/doi/full/10.1002/we.2785), we tuned the parameter `k_vCurl` so as to better match between the curled and polar wake solutions for a rotor without skewed inflow, but this calibration is likely case dependent. 

Are you using default values for the curled-wake parameters?  Can you change `k_vCurl` so as to better match the curl and polar wake solutions for a case without skewed inflow?

Best regards,

**Reply by MYMahfouz** [reply]
Dear @jjonkman,

Thanks for your answer.

The results I showed are also without skewed inflow.

I am using the `DEFAULT` value of  `k_vCurl` which is equal to `2`. A far as I understood from the paper I need to increase this value to `2.7` or `3`. I am running the simulations just below rated wind speed so the thrust coefficient is still high.

Am I correct? do you have different suggestions for the `k_vCurl` values?

Best regards,

**Reply by jjonkman** [reply, marked as answer]
Dear @MYMahfouz,

Again, you may have to tune `k_vCurl` for your specific case.  The higher the value of `k_vCurl`, the higher the eddy viscosity will be, resulting in faster wake recovery (weaker wake deficits).

Best regards,

**Reply by MYMahfouz** [reply]
Dear @jjonkman,

Thank you. I tuned it and the two models match much better at `k_vCurl=6`. 

Best regards,

**Reply by MYMahfouz** [reply]
Dear @jjonkman, 

Sorry to bother you again with this question.

Should i retune the model if the turbine is floating or fixed bottom? What would be the reference to retune it?

What exactly does the `k_vCurl` value change in the velocity deficit calculation? I understand from the paper you mentioned that it calibrates the model and if i understand properly higher value increases diffusion. But how is this affected by the motion of the rotor in case of a floating wind turbine?

Best regards,

**Reply by jjonkman** [reply]
Dear @MYMahfouz,

NREL has calibrated the tuning and semiempirical parameters of FAST.Farm against large-eddy simulation (see the related papers), and we are hopeful these parameters are generally reasonable for other wind turbines and/or conditions.  But if your wind turbine or conditions are quite different from the cases used to tune the parameters, it is certainly useful for more accurate results from FAST.Farm to recalibrate these parameters.  Of course, doing so takes work (e.g., following the process outlined in the papers), as well as data (high-fidelity modeling results or experimental measurements) that you trust to calibrate against.

For a clear definition of `k_curl`, see the "Eddy viscosity model" section of our _Wind Energy_ journal article explaining how the curled wake model was introduced in FAST.Farm: https://onlinelibrary.wiley.com/doi/full/10.1002/we.2785.

I would say that the wake diffusion in the wake models currently available in FAST.Farm is not strongly tied to the motion of a floating wind turbine.  The motion of the floating wind turbine mainly effects the location where wakes are "released" from the rotor (this location oscillates with the motion of the floating turbine) and the deflection of the wake (e.g., due to platform pitch).  NREL's first presentation on the application of FAST.Farm to floating wind turbines--including validation against large-eddy simulation and application to a small wind floating wind farm) will be presented at the upcoming NAWEA / WindTech 2023 conference, which will be followed shortly after by a journal publication.

Best regards,

**Reply by MYMahfouz** [reply]
Dear @jjonkman,

Thank you for your reply. 
I think the curl model needs more calibration than just `k_curl`, I am not sure but this is what I think when i see the wake deficit results. 

I am looking forward to reading the paper. So please let me know whenever it is publicly available.

Best regards,
Youssef
